go.mod

```diff
@@ -218,6 +218,8 @@ require (
 	sigs.k8s.io/yaml v1.4.0 // indirect
 )
 
+require github.com/openshift-kni/numaresources-operator v0.1.1
```

Comment:
from what I see, the v0.1.1 is 3 years old. 

Don't you want to use the one tied with OCP Version ?

Reply:
yep, good idea, thanks!